hadoop-hdds/docs/content/feature/ContainerBalancer-cfg.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Configuring container balancer service"
+weight: 12
+menu:
+   main:
+      parent: "Container Balancer overview"
+summary: To use container balancer using Cloudera Manager, perform the following steps.
```

Comment:
This file shouldn't be used as it only provides Cloudera specific steps and information